androdeb

```diff
@@ -66,6 +67,12 @@ case $key in
 esac
 done
 
+if [ ! -z "$KERNELSRC" -a ! -z "KBUILDOUT" ];
```

Comment:
I prefer if its called "KERNELOUT" to be consistent with "KERNELSRC"

Reply:
I'll change it to be "KERNELOUT" as you suggest.
What's about the '--kbuildout'? Do we also need change it to be "--kernelout"?